src/main/java/org/jenkinsci/plugins/badge/PublicBadgeAction.java

```diff
@@ -95,35 +95,41 @@ public String getDisplayName() {
     public HttpResponse doCoverageIcon(StaplerRequest req, StaplerResponse rsp, @QueryParameter String job, @QueryParameter String style) {
     	Job<?, ?> project = getProject(job);
         Integer codeCoverage = null;
-        
+
         // Checks for Cobertura
         if (project.getLastSuccessfulBuild() != null) {
-	        CoberturaBuildAction coverageAction = project.getLastSuccessfulBuild().getAction(CoberturaBuildAction.class);
-	        if(coverageAction != null){
-	          codeCoverage = new Integer(coverageAction.getBuildHealth().getScore());
-	        }
+	        if (Jenkins.getInstance().pluginManager.getPlugin("cobertura").isActive()) {
```

Comment:
[Checkstyle] WARNING: 'if' have incorrect indentation level 16, expected level should be 6.